python/lsst/meas/algorithms/interp.cc

```diff
@@ -34,32 +34,32 @@ namespace algorithms {
 
 namespace {
 template <typename PixelT>
-void declareInterpolateOverDefects(py::module& mod) {
+static void declareInterpolateOverDefects(py::module& mod) {
     mod.def("interpolateOverDefects",
             interpolateOverDefects<lsst::afw::image::MaskedImage<PixelT, lsst::afw::image::MaskPixel,
                                                                  lsst::afw::image::VariancePixel>>,
             "image"_a, "psf"_a, "badList"_a, "fallBackValue"_a = 0.0, "useFallbackValueAtEdge"_a = false);
 }
 }
 
-PYBIND11_PLUGIN(_interp) {
-    py::module mod("_interp", "Python wrapper for afw _interp library");
+PYBIND11_PLUGIN(interp) {
+    py::module mod("interp");
 
     py::class_<Defect, std::shared_ptr<Defect>, lsst::afw::image::DefectBase> clsDefect(mod, "Defect");
 
     /* Member types and enums */
     py::enum_<Defect::DefectPosition>(clsDefect, "DefectPosition")
-        .value("LEFT", Defect::DefectPosition::LEFT)
-        .value("NEAR_LEFT", Defect::DefectPosition::NEAR_LEFT)
-        .value("WIDE_LEFT", Defect::DefectPosition::WIDE_LEFT)
-        .value("MIDDLE", Defect::DefectPosition::MIDDLE)
-        .value("WIDE_NEAR_LEFT", Defect::DefectPosition::WIDE_NEAR_LEFT)
-        .value("WIDE", Defect::DefectPosition::WIDE)
-        .value("WIDE_NEAR_RIGHT", Defect::DefectPosition::WIDE_NEAR_RIGHT)
-        .value("NEAR_RIGHT", Defect::DefectPosition::NEAR_RIGHT)
-        .value("WIDE_RIGHT", Defect::DefectPosition::WIDE_RIGHT)
-        .value("RIGHT", Defect::DefectPosition::RIGHT)
-        .export_values();
+            .value("LEFT", Defect::DefectPosition::LEFT)
+            .value("NEAR_LEFT", Defect::DefectPosition::NEAR_LEFT)
+            .value("WIDE_LEFT", Defect::DefectPosition::WIDE_LEFT)
+            .value("MIDDLE", Defect::DefectPosition::MIDDLE)
+            .value("WIDE_NEAR_LEFT", Defect::DefectPosition::WIDE_NEAR_LEFT)
+            .value("WIDE", Defect::DefectPosition::WIDE)
+            .value("WIDE_NEAR_RIGHT", Defect::DefectPosition::WIDE_NEAR_RIGHT)
+            .value("NEAR_RIGHT", Defect::DefectPosition::NEAR_RIGHT)
+            .value("WIDE_RIGHT", Defect::DefectPosition::WIDE_RIGHT)
+            .value("RIGHT", Defect::DefectPosition::RIGHT)
+            .export_values();
```

Comment:
Is it `clang-format` that wants this extra indentation for chained method calls?